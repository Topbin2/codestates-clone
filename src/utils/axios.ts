import axios from "axios";

export const custom = axios.create({
  baseURL: "https://jsonplaceholder.typicode.com",
});
