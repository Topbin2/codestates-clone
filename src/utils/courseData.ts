export const courseData = [
  {
    id: 1,
    url: "https://urclass.codestates.com/static/media/img-cover.23e06f01.jpg",
    title: "[SEB FE] Section 1",
    body: "웹 프론트엔드 개발의 기초지식을 기반으로 스스로 단순한 Web App을 만들 수 있다.",
  },
  {
    id: 2,
    url: "https://urclass.codestates.com/static/media/img-cover.23e06f01.jpg",
    title: "[SEB FE] Section 2",
    body: "서버와 통신이 가능한 구조적인 Web App을 만들 수 있다.",
  },
  {
    id: 3,
    url: "https://urclass.codestates.com/static/media/img-cover.23e06f01.jpg",
    title: "[SEB FE] Section 3",
    body: "사용자 친화적이고 안전한 Web App을 만들 수 있다.",
  },
  {
    id: 4,
    url: "https://urclass.codestates.com/static/media/img-cover.23e06f01.jpg",
    title: "[SEB FE] Section 4",
    body: "사람과 기계가 모두 쉽고 빠르게 접근 가능한 Web App을 만들 수 있다.",
  },
  {
    id: 5,
    url: "https://urclass.codestates.com/static/media/img-cover.23e06f01.jpg",
    title: "[SEB FE] Project",
    body: "First Project, Final Project",
  },
  {
    id: 6,
    url: "https://urclass.codestates.com/static/media/img-cover.23e06f01.jpg",
    title: "[SEB FE] Wiki & Survey",
    body: "SEB FE 부트캠프에서 활용되고 있는 Wiki & Survey",
  },
];
