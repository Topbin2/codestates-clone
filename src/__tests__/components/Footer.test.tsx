import renderer from "react-test-renderer";
import Footer from "../../components/Footer";
import { render } from "../../utils/test-utils";

describe("Footer", () => {
  it("snapshot", () => {
    const component = renderer.create(<Footer />);
    expect(component.toJSON()).toMatchSnapshot();
  });

  it("span 태그는 해당 텍스트를 포함한다.", () => {
    const { container } = render(<Footer />);
    const span = container.querySelector("span");
    expect(span).toHaveTextContent("© Copyright ⓒ 2022 CodeStates");
  });
});
