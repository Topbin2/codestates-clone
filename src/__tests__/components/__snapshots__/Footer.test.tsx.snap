// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<div
  className="footer"
>
  <div
    className="sc-bczRLJ lnbbFP"
  >
    <div
      className="footer_bottom"
    >
      <span>
        © Copyright ⓒ 2022 CodeStates
      </span>
    </div>
  </div>
</div>
`;
